### PR TITLE
docs: website package internal only

### PR DESCRIPTION
**Pull request by benomatis**
Added a note that the website package is intended for internal use only.

See Slack discussion https://contentful-community.slack.com/archives/C5AJ45S6R/p1667895043907169?thread_ts=1667658184.124739&cid=C5AJ45S6R

### CODE REVIEW

File: packages/website/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Forma36 Documentation Website
 
+## Note to outside contributors
+
+We did not expect outside contributors to run the forma documentation website locally, as this content is coming from the ui-team's contentful space, and is intended for internal use. For sandboxing and playing around locally we recommend to start the storybook package locally (see [the 'Storybook for f36-components' section on the main page](https://github.com/contentful/forma-36#storybook-for-f36-components)).
```
Comment (ghepting) [inline packages/website/README.md:5]: I might recommend this be worded as follows:

```
## The forma documentation site does not currently support being run locally by contributors outside of Contentful internal UI systems team as it has a dependency on the team's private Contentful space. Community contributors can run storybook to build and develop components locally (see [the 'Storybook for f36-components' section on the main page](https://github.com/contentful/forma-36#storybook-for-f36-components)).
```

Comment (benomatis) [reply packages/website/README.md:5]: looks great, thanks!

**Review by Lelith — APPROVED** [review]
thank you for your contribution

**Review by massao — APPROVED** [review]

**Comment by bgutsol** [comment]
Had to replicate the PR https://github.com/contentful/forma-36/pull/2343 due to failing CI. We will try to figure out why CI doesn't work properly with external contributions.

Nevertheless thanks a lot for your contribution ❤️

**Pull request closed without merging**